network/api/v1/api.go

```diff
@@ -76,14 +81,14 @@ func (a Wrapper) GetTransaction(ctx echo.Context, hashAsString string) error {
 func (a Wrapper) GetTransactionPayload(ctx echo.Context, hashAsString string) error {
 	hash, err := hash2.ParseHex(hashAsString)
 	if err != nil {
-		return ctx.String(http.StatusBadRequest, err.Error())
+		return core.NewProblem(problemTitleGetTransactionPayload, http.StatusBadRequest, err.Error())
 	}
 	data, err := a.Service.GetTransactionPayload(hash)
 	if err != nil {
-		return ctx.String(http.StatusInternalServerError, err.Error())
+		return core.NewProblem(problemTitleGetTransactionPayload, http.StatusInternalServerError, err.Error())
 	}
 	if data == nil {
```

Comment:
Logging Is not part of the PR, but shouldn't an internal server error be logged?

Reply:
Agreed. I must admit I spotted this as well but was kinda lazy. Will add a log statement.